src/components/Alert/Alert.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 
+import { ThemeProvider } from 'styled-components';
 import { StyledAlert } from './Alert.styled';
 
-const Alert = props => {
-    return (
-        <StyledAlert>{props.children}</StyledAlert>
-    );
-}
+const Alert = (props) => {
+  return (
+    <ThemeProvider theme={props.theme || {}}>
```

Comment:
Rozumiem, że to zadanie itp. ale docelowo Provider definiujemy gdzieś na górze hierarchii komponentów, aby wygodnie stylować przepływem `theme`  - to tak na przyszłość